### PR TITLE
Replace patch and UI icons with SVGs

### PR DESCRIPTION
**Pull request by HexapodPhilosopher**
**Brief Description of What This PR Does**

Adds new .svg icons for spinboxes, optionButtons and sliders.

Replaces patch icons with SVGs.

Re-scales compound .svg icons to 256x256.

**Related Issues**

Fixes some parts of #1581 :
   * spinbox up/down arrows...
   * slider circular part (grab.png with all variants and sliderTick.png)
   * all of the patch map, patch icons...

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [X] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [x] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
Could you clarify which parts of #1581 this fixes so that when I have a look at this tomorrow it'll be easier to focus my testing?

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
There's a few concrete things I think should be fixed. Additionally I noticed that 4 patch icons now have border artifacts when zoomed out in the map fully: 

kuva


If I reduce the resolution limit to 128 for the cave patch it seems to mostly get rid of the problem but not entirely, so that might be a doable stop-gap measure.

Also this pretty nicely improves the look of the compound icons so that is nice, but there is still ever so slight border artifacts on the iron icon. I tried reducing the size to 128x128 and that maybe kind of helped a bit but made the iron look a bit blurrier than other icons. So for now I think the icon tweaking is good enough.

I also wanted to say that it looks like SVGs have quite many pitfalls about their use in Godot, but I hope we can solve them enough that no players will notice it.

File: assets/textures/gui/bevel/SliderTick.svg
[the remark this answers is not in the document]
Comment (HexapodPhilosopher) [reply assets/textures/gui/bevel/SliderTick.svg:1]: No, it's not centered. I had some trouble getting it to the right size, and didn't notice the misalignment. I'll fix that.

File: assets/textures/gui/bevel/Grab.svg
[the remark this answers is not in the document]
Comment (HexapodPhilosopher) [reply assets/textures/gui/bevel/Grab.svg:1]: I actually made them opaque to start with, then noticed the previous ones were slightly transparent, so was trying to replicate that.

[the remark this answers is not in the document]
Comment (hhyyrylainen) [reply assets/textures/gui/bevel/Grab.svg:1]: The previous ones surely weren't *as* transparent at least I don't remember constantly being able to see the underlying bar even when not moving the slider around, right?

**Comment by HexapodPhilosopher** [comment]
I've reduced the blurring of the corners and edge of the icons by increasing the base sizes and scaling down in the editor. The compound icon blur is not even visible to me anymore. I also removed a few disused ui textures.

**Review by hhyyrylainen — APPROVED** [review]
I think all the artifacts I saw are now gone, and the slider tick is correctly aligned. So good job.

Let's remember for the future that tweaking the SVG scale to be less than 1 while increasing the SVG size (and setting a maximum size in Godot) is what seems to fix the artifacting problems.

**Pull request merged**